extension/src/plots/webview/messages.ts

```diff
@@ -61,18 +61,12 @@ export class WebviewMessages {
   }
 
   public async sendWebviewMessage() {
-    const selectedRevisions = this.plots.getSelectedRevisionDetails()
-
-    await this.getWebview()?.show({
-      cliError: this.errors.getCliError()?.error || null,
-      comparison: this.getComparisonPlots(),
-      custom: this.getCustomPlots(),
-      hasPlots: !!this.paths.hasPaths(),
-      hasUnselectedPlots: this.paths.getHasUnselectedPlots(),
-      sectionCollapsed: this.plots.getSectionCollapsed(),
-      selectedRevisions,
-      template: this.getTemplatePlots(selectedRevisions)
-    })
+    const webview = this.getWebview()
```

Comment:
[F] although this function is duplicated it is not worth the hassle that a base class would bring